### DISCUSSION TITLE
Switch between collections

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#15142] [Ideas]
Maybe add a keyboard shortcut similar to the one in the stable version - shift + left/right?
I cannot navigate to the next collection using only the keyboard

**Comment by smoogipoo** [comment]
This needs collections to show up as groups in song select first.

**Reply by peppy** [reply]
Does it? I think this is a request to change the active collection while only one is shown. Would need confirmation on what the goal here is.

**Reply by smoogipoo** [reply]
If we added shift+left/right just to change collections, that would be an extremely hidden shortcut imo. Also I don't see why normal groupings wouldn't have the option to move between groupings with the same command.

I assume the "goal" here is to work like stable does, shift+left/right moves between groupings (which also works for any grouping mode, fwiw).

**Reply by peppy** [reply]
Right, sounds a lot better in that case.
